modules/installation-user-infra-machines-advanced-customizing-live-iso-ca-certs.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+[id="installation-user-infra-machines-advanced-customizing-live-iso-ca-certs_{context}"]
+= Modifying a live install image while using a custom certificate authority
+
+You can provide CA certificates to both the installation boot and the installed system with the `--ignition-ca` flag of the `customize` subcommand.
+
+.Procedure
+
+. Download the `coreos-installer` binary from the link:https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/[`coreos-installer` image mirror] page.
+
+. Retrieve the {op-system} ISO image from the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/[{op-system} image mirror] page and run the following command to customize the ISO image for use with a custom CA:
++
+[source,terminal]
+----
+$ coreos-installer iso customize rhcos-<version>-live.x86_64.iso --ignition-ca cert.pem
+----
++
+[NOTE]
+====
+Custom CA certificates affect how Ignition fetches remote resources but do not affect the certificates installed onto the system.
+====
++
+Your CA certificate is applied the next time you boot the ISO image.
```

Comment:
Again, this could imply that the application happens at a point in time, when in fact it happens on every boot of the customized ISO.